src/main.rs

```diff
@@ -38,6 +38,7 @@ fn run() -> i32 {
     opts.optopt("i", "id", "Window to capture", "ID");
     opts.optopt("g", "geometry", "Area to capture", "WxH+X+Y");
     opts.optopt("f", "format", "Output format", "png/pam");
+    opts.optflag("p", "print", "Print filename on stdout");
```

Comment:
The long name for this option should be something more explicit, such as `--print-path`.

Reply:
Sure, fixed.